pages/_en-US/ds-index/special-games.md

```diff
@@ -310,3 +310,19 @@ Currently, nds-bootstrap does not have Slot-2 emulation. Included are some possi
 - Personal Trainer: Walking -> Mii Channel
 - Ultimate Band -> Ultimate Band (Wii)
 - WarioWare D.I.Y. -> WarioWare D.I.Y. Showcase
+
+### Other (Carts that have features that don't fall into an above category) 
+Learn with Pokemon: Typing Adventures 
+ -Cartridge has special bluetooth chip for connecting to the keyboard
+Pokemon HG/SS
+ -Cartridge Has IR port for connecting to the PokeWalker
+ ElecktroPlankton
+  -Cartridge has no save RAM
+ DS Download station
+  -Series of cartridges used to distribue demo games
+ Sonic Classics Collection
+  -Genesis emulation structure has been converted into an emulator used in TWiLight Menu++
+ Nitro Development Cart
+  - Has extra ram for development of games for the Nintendo DSi
```

Comment:
fyi you need a space after the `-` for it to work as a bullet and sub bullets should be indented by three spaces. You should probably also make the base level ones be bullets. Like this or so:
```suggestion
- Learn with Pokemon: Typing Adventures 
   - Cartridge has special bluetooth chip for connecting to the keyboard
- Pokemon HG/SS
   - Cartridge Has IR port for connecting to the PokeWalker
- ElecktroPlankton
   - Cartridge has no save RAM
- DS Download station
   - Series of cartridges used to distribue demo games
- Sonic Classics Collection
   - Genesis emulation structure has been converted into an emulator used in TWiLight Menu++
- Nitro Development Cart
   - Has extra ram for development of games for the Nintendo DSi
```